dl/pom.xml

```diff
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_${scala.major.version}</artifactId>
-            <scope>test</scope>
+            <scope>${spark-scope}</scope>
```

Comment:
Agree, scalatest shouldn't be anything but test scope.
There is a particular reason it's _not_ in one case in Spark itself, but, it's specific to Spark.